portal/static/styles.css

```diff
@@ -38,26 +46,425 @@ body {
+  position: relative; 
+  cursor: pointer; 
+  font-size: 20px; 
+} 
+  
+/* Hide the default checkbox in class.html */ 
+.courseCheckbox input[type=checkbox] { 
+    visibility: hidden; 
+    width: 0;
+    height: 0;
+} 
+  
+/* Creating a custom checkbox 
+based on demand */ 
+.fakeCheckbox { 
+    position: absolute; 
+    top: 0; 
+    left: 0; 
+    height: 25px; 
+    width: 25px; 
+    border-right: 1px solid white;
+    border-bottom: 1px solid white;
+    border-radius: 10px 0 10px 0;
+} 
+
+.courseCheckbox input {
+  width: 50%;
+}
+  
+/* Specify the background color and border to be 
+shown when hovering over checkbox */ 
+.courseCheckbox:hover input ~ .fakeCheckbox { 
+  background-color: white; 
+  border: 1px solid #a6083d;
+} 
+
+/* Specify the background color to be
+shown when hovering over checkbox*/
+.courseCheckbox:active input ~ .fakeCheckbox {
+  background-color: #ebebeb;
+}
+  
+/* Specify the background color and border to be 
+shown when checkbox is checked */ 
+.courseCheckbox input:checked ~ .fakeCheckbox { 
+  background-color: #a6083d;
+  border-right: 1px solid white;
+  border-bottom: 1px solid white;
+   
+} 
+  
+/* Checkmark to be shown in checkbox */ 
+/* It is not be shown when not checked */ 
+.fakeCheckbox::after { 
+    content: ""; 
+    position: absolute; 
+    display: none; 
+} 
+  
+/* Display checkmark when checked */ 
+.courseCheckbox input:checked ~ .fakeCheckbox::after { 
+    display: block; 
+} 
+  
+/* Styling the checkmark using webkit */ 
+/* Rotated the rectangle by 45 degree and  
+showing only two border to make it look 
+like a tickmark */ 
+.courseCheckbox .fakeCheckbox::after { 
+    left: 8px; 
+    bottom: 5px; 
+    width: 6px; 
+    height: 12px; 
+    border: solid white; 
+    border-width: 0 4px 4px 0; 
+    -webkit-transform: rotate(45deg); 
+    -ms-transform: rotate(45deg); 
+    transform: rotate(45deg); 
+} 
+
+/***Styling Edit courses button***/
+
+.course button, .createSession button, .editSession button, .submitCourse {
+  display: block;
+  margin: 1rem auto;
+  background-color: white;
+  color: #a6083d;
+  border: 1px solid white;
+  padding: .5rem 1rem;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.course button:hover, .createSession button:hover, .editSession button:hover, .submitCourse:hover {
+  background-color: #a6083d;
+  color: white;
+}
+
+
+/***Create a Session***/
+
+.createSession, .editSession {
+  align-self: start;
+  color: white;
+  background-color: #a6083d;
+  width: 50%;
+  margin: 1rem;
+  padding: 2rem 2rem 0 2rem;
+  border: 1px solid black;
+  border-radius: 10px;
+}
+
+.createSession ul, .editSession ul {
+  display: flex;
+  justify-content: start;
+  padding: 0;
+  list-style-type: none;
+}
+
+.createSession ul li, .editSession ul li {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 1rem;
+}
+
+.createSession input, .editSession input {
+  color: black;
+  margin-right: 1rem;
+}
+
+/***Styling Roster checkboxes***/
```

Comment:
A lot of this css is identical to the other checkbox styling above and should be consolidated